public/css/style.css

```diff
@@ -706,13 +707,31 @@ html {
 .hidden {
   display: none;
 }
-
 .query {
   padding: 0rem;
   background-color: #9753e1;
   position: relative;
   min-height: 500px;
 }
+.query--no-results-wrapper {
+  width: 100%;
+  text-align: center;
+}
+.query--no-results {
+  width: 70%;
+  color: #9753e1;
+  height: 12rem;
+  white-space: nowrap;
+  overflow: hidden;
```

Comment:
probably can just do `overflow-x: hidden` here then remove the `overflow-y: scroll` below. `scroll` says "always show scrollbar, even if i don't need it"